labgrid/pytestplugin/hooks.py

```diff
@@ -130,3 +130,22 @@ def pytest_collection_modifyitems(config, items):
                     reason=f'Skipping because features "{missing_feature}" are not supported'
                 )
             item.add_marker(skip)
+
+@pytest.hookimpl(wrapper=True)
```

Comment:
Why is this a [hook wrapper](https://docs.pytest.org/en/stable/how-to/writing_hook_functions.html#hook-wrappers-executing-around-other-hooks)? It doesn't process/return the `yield`.

[`tryfirst=True`](https://docs.pytest.org/en/stable/how-to/writing_hook_functions.html#hook-function-ordering-call-example) should be enough.